### ISSUE TITLE
Check that pages aren't using H1 headers at the top level

### ISSUE THREAD

**Issue by jen-machin** [dfe-analytical-services/analysts-guide#85]
**Describe the bug**
The title of each page is intended to act as the H1 level header, see this discussion: https://github.com/quarto-dev/quarto-cli/issues/3891

https://github.com/quarto-dev/quarto-cli/issues/3891

Any instances where H1 headers (using just one `#`) are being used should be changed to H2 headers instead for accessibility purposes. Each page on the site will need to be checked.
